apps/studio/components/interfaces/Settings/General/Infrastructure/PauseProjectButton.tsx

```diff
@@ -31,8 +32,11 @@ const PauseProjectButton = () => {
     PermissionAction.INFRA_EXECUTE,
     'queue_jobs.projects.pause'
   )
+  const isOrioleDbInAws = useIsOrioleDbInAws()
+  const isOrioleDb = useIsOrioleDb()
```

Comment:
suspect this proxy might be a fragile test, but i guess the impact of breakage is pretty small